### PR TITLE
Zipcode input screen

### PR DESCRIPTION
**Pull request by kyi193**
I created the zip code input screen page that the user will first see when they open the app. The zip code is stored into the Redux store which will be used in the GestureSpinnerWheel component after the user clicks submit

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -12,7 +13,8 @@ export default function App() {
   return (
     <Provider store={createStore(reducer, composeWithDevTools(middleware))}>
       <View style={styles.container}>
-        <GestureSpinnerWheel />
+        {/* <GestureSpinnerWheel /> */}
```
Comment (squidapillar) [inline App.js:16]: remove comment

File: App.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { StatusBar } from 'expo-status-bar';
 import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 import GestureSpinnerWheel from './components/GestureSpinnerWheel'
```
Comment (squidapillar) [inline App.js:4]: remove `GestureSpinnerWheel` import and remove comment

Comment (kyi193) [reply App.js:4]: Sure thing

Comment (kyi193) [reply App.js]: I added some unit testing for this specific component. I'lll add more unit testing for the spinner wheel next on a separate feature branch where I connect GestureSpinnerWheel to the store and get the wheel the actually render random restaurants based on the the zip code stored in Redux.

**Pull request merged**